Editor/Window/ConnectToFleetInput.cs

```diff
@@ -120,15 +135,19 @@ private async Task UpdateFleetMenu()
         {
             if (_stateManager.GameLiftWrapper != null)
             {
-                var fleetList = await _fleetManager.ListFleetAttributes();
-                if (fleetList == null)
+                _fleetAttributes = await _fleetManager.DescribeFleetAttributes();
+                if (_fleetAttributes == null)
                 {
-                    _fleetAttributes = new List<FleetAttributes>();
+                    var url = string.Format(Urls.AwsGameLiftLogs, _stateManager.Region);
+                    _connectToAnywhereStatusBox.Show(StatusBox.StatusBoxType.Error, Strings.AnywherePageStatusBoxDefaultErrorText, "", externalButtonLink: url, externalButtonText:Strings.ViewLogsStatusBoxUrlTextButton);
```

Comment:
I think `AnywherePageStatusBoxDefaultErrorText` may not be appropriate to this case, right? The case is that no fleets exists in the customers account?

Reply:
Anything that could make the manager throw an error could trigger this to occur. So there is multiple errors that could be thrown, so we account for all with this.

Reply:
That's fair, but this doesn't correspond to an error right? You are querying for the fleet list and there just doesn't happen to be any fleets in the customers account. This happens in the UpdateFleetMenu method in the constructor.

The "additionalText" is also "", which doesn't help convey what the customer is expected to do next for things to work.

Reply:
So I have looked into the docs and managed to try for errors. If the user doesn't have any fleets, it will not catch in our try catch. So it will just return an empty `List<FleetAttributes>` meaning that it will not hit the error box.